In_Db2_Machine_Learning/In DB Model Deployment and Management/README.md

```diff
@@ -0,0 +1,337 @@
+# Procedure for using db2 as a backend for model hosting
+
+DB2's Integrated Analytics System allows for an impressive amount of user defined extensions, which can be leveraged to allow db2 to serve as a model management and scoring engine. 
+We can store, manage, update and score models trained in scikit-learn and other python frameworks in DB2. We can also 
+run models trained in Lua, C, R C++. This document will demonstrate a python based example. 
+
+# DB2 Configuration
+
+We will need to store a python class for scoring a generic model in our DB2 server, and make some minor configuration changes. 
+
+Log in to the db2 server. In this case the user is `db2inst1`
+
+`source ~/.bashrc; sudo su - db2inst1`
+
+Make sure there is a python3 installation in your server.
+
+`sudo yum install python3`
+
+Install scikit-learn and joblib
+
+`pip3 install scikit-learn joblib`
+
+Update db2 to use the correct python path
+
+`db2 update dbm cfg using PYTHON_PATH $(which python)`
+
+Restart the database
+
+`db2 connect reset;`
+`db2stop`
+`db2start`
+
+Add the python class for scoring the model by writing the following python class to $DB2_HOME/function/routine/score.py
+
+
+```
+import nzae
+from joblib import load
+from io import BytesIO
+import base64
+
+class predict(nzae.Ae):
+    def _setup(self):
+        self.model = None
+
+    def predict(self,data):
+        model = data[0]
+        if not self.model:
+            self.model = load(BytesIO(base64.b64decode(model)))
+
+        data = data[1:]
+        result = self.model.predict([data])
+        return float(result[0])
+
+    def _getFunctionResult(self,row):
+        price = self.predict(row)
+        return price
+
+predict.run()
+```
+
+## Model Training
+
+We will first train a simple logistic regression model on the Iris data set. We will use joblib to pickle th
```

Comment:
```suggestion
We will first train a simple logistic regression model on the Iris data set. We will use joblib to pickle the
```

Not a problem, I'm just curious if there are any comments you'd want to add about why `joblib` is being used instead of `pickle` itself.  I've seen both used, it might just be helpful to future readers of this to understand why.